docs/slack.rst

```diff
@@ -46,3 +48,45 @@ Chances are we have a large group of folks in your neck of the woods,
 so join up and chat with them.
 They're a great way to get in touch with your local meetup organizer and community.
 
+
+Social Rules
+------------
+
+Our goal with Write the Docs is to be **the most welcoming community on the internet**.
+Documentarians tend to be empathetic people,
+this means that we should be good at communicating well.
+People who are new to our community in particular should be assisted and welcoming.
+
+This concept is inspired by the `Recurse Center's Social Rules <https://www.recurse.com/manual#social-rules>`_ -- which you should also follow.
```

Comment:
I find the 

> which you should also follow 

kinda confusing. Surely better to take enough from there so people don't have to read both. Also OMG long!